containers/daap-presigned-url/tests/unit/mock_test.py

```diff
@@ -44,7 +49,7 @@ def test_success(s3_client, fake_context, region_name, monkeypatch):
     assert body["URL"]["url"] == "https://bucket.s3.amazonaws.com/"
     assert (
         body["URL"]["fields"]["key"]
-        == f"raw/database1/v1.0/table1/load_timestamp=20230101T000000Z/{a_uuid}"
+        == f"raw/database1/v1.0/table1/load_timestamp=20230101T000000Z/{a_uuid}{os.path.splitext(filename)[1]}"
```

Comment:
See pathlib comment above - if implementing that way, likely want to reflect here.

I would make this more explicit for the expectation, potentially by splitting out the filename into filename + extension above
```suggestion
... filename = "testdata"  # defined at the top of the function
... extension = ".csv"
... file_ = filename + extension
        == f"raw/database1/v1.0/table1/load_timestamp=20230101T000000Z/{a_uuid}{extension}"
```

Reply:
Now using PurePath and new file_extension variable has ben created and referenced in the assertion.